Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rspec'
+gem 'capybara'
+gem 'json'
+gem 'byebug'
+gem 'rspec-sinatra'
```

Comment:
Where possible, try to separate your test and development dependencies in your Gemfile